packages/stark-testing/package-lock.json

```diff
@@ -5,95 +5,95 @@
   "requires": true,
   "dependencies": {
     "@babel/code-frame": {
-      "version": "7.5.5",
```

Comment:
This `package-lock.json` file doesn't need to be updated since there is no `@types/lodash` in here... could you remove this file from this PR?